### DISCUSSION TITLE
Add page number as an argument to execute_command_a

### DISCUSSION THREAD

**Discussion by btrkeks** [ahrm/sioyek#898] [Ideas]
It would be cool if you could add an argument %4 which would expand to the current page number for execute_command. I am currently using zathura to make flash cards from books on key press and being able to have the page number filled in automatically is the only feature that's slightly holding me back.

Passing the current theorem / definition number would be even better, but I guess that's not feasible.

**Comment by ahrm** [comment]
We already do that? See here:
https://github.com/ahrm/sioyek/blob/779b52e36de9d596dc708ada045ea48af0c21c9d/pdf_viewer/main_widget.cpp#L2289

**Reply by btrkeks** [reply]
Oh, perfect. Then I guess the documentation needs an update. I was looking at https://sioyek-documentation.readthedocs.io/en/latest/configuration.html
